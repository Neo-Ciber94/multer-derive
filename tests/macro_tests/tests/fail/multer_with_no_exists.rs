use multer_derive::FromMultipart;

#[derive(FromMultipart)]
struct MyStruct {
    #[multer(with = "not_exists")]
    text: Text,
    number: u32,
}

struct Text(String);

fn main() {}
