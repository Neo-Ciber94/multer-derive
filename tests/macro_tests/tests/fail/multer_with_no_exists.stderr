error[E0425]: cannot find function `not_exists` in this scope
 --> tests/fail/multer_with_no_exists.rs:3:10
  |
3 | #[derive(FromMultipart)]
  |          ^^^^^^^^^^^^^ not found in this scope
  |
  = note: this error originates in the derive macro `FromMultipart` (in Nightly builds, run with -Z macro-backtrace for more info)
